coffee_maker.py

```diff
@@ -0,0 +1,29 @@
+class CoffeeMaker:
```

Comment:
Nice use of classes.

Reply:
thank you. I'm still getting familiar with them